api/admin/controller.py

```diff
@@ -1675,15 +1677,15 @@ def _get_integration_protocols(self, provider_apis, protocol_name_attr="__module
                 protocol["sitewide"] = sitewide
 
             settings = getattr(api, "SETTINGS", [])
-            protocol["settings"] = settings
+            protocol["settings"] = list(settings)
```

Comment:
This was necessary because the list may be modified later by the insertion of the 'mirror integration' setting. We can't modify the class SETTINGS object because the list of storage integrations might change later.